meta-mel/conf/local.conf.append.ls1021atwr

```diff
@@ -0,0 +1,2 @@
+#LS1021atwr supports usb-audio & simple soundcard, alsa utils are required for playback/record operations
```

Comment:
I hate to nitpick here, but this doesn't line up with all the other comments in the file. We wrap long lines, and use a space after # :)